src/app/searchBar.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState, useEffect } from "react";
+import { AutoComplete } from "antd";
+import { SearchOutlined } from "@ant-design/icons";
+import { Button } from "antd";
+
+const SearchBar: React.FC<{
+  search: string;
+  setSearch: any;
```

Comment:
Those names are quite ambigous.

Maybe something like: `searchCallback`? or `onSearchCallback`.

You are quite leaking implementation details here. This component shouldn't be awere of the fact that other compoentn has `search` state variable